src/openloco/ui/WindowManager.cpp

```diff
@@ -1824,6 +1824,93 @@ namespace openloco::ui::WindowManager
         }
     }
 
+    /**
+     * 0x004A0A18
+     *
+     * @param flags @<al>
+     */
+    void viewportSetVisibility(uint8_t flags)
```

Comment:
This isn't really a `flags` but I don't know what a good name for it would be. It should ideally become an enum class with 4 different options.